2025-09-05 10:23:45 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 10:24:57 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 11:15:09 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 11:15:21 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 11:15:35 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 11:16:09 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 11:20:01 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:07:04 [error]: 500 - next is not defined {
  "url": "/api/dashboard/stats",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:07:32 [error]: 500 - next is not defined {
  "url": "/api/dashboard/activities",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:07:37 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:16:12 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:16:12 [error]: 500 - next is not defined {
  "url": "/api/dashboard/stats",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:16:14 [error]: 500 - next is not defined {
  "url": "/api/dashboard/activities",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:21:54 [error]: 500 - next is not defined {
  "url": "/api/calendrier",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:22:02 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:23:11 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:42:44 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:42:49 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:42:52 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:42:58 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:43:00 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:50:46 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:51:04 [error]: 500 - next is not defined {
  "url": "/api/users/me/settings",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:51:48 [error]: 500 - next is not defined {
  "url": "/api/users/me/settings",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:51:54 [error]: 500 - next is not defined {
  "url": "/api/users/me/settings",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:52:06 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:52:14 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:52:18 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:56:28 [error]: 500 - next is not defined {
  "url": "/api/calendrier",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 17:56:59 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:04:40 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:06:21 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:17:05 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:24:29 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:31:12 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:31:21 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:31:24 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:31:26 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:31:46 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:31:46 [error]: 500 - next is not defined {
  "url": "/api/calendrier",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:35:19 [error]: 500 - next is not defined {
  "url": "/api/calendrier",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:35:26 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:35:51 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:35:53 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:35:54 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:35:56 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:35:58 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:36:07 [error]: 500 - next is not defined {
  "url": "/api/users/me/settings",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:36:12 [error]: 500 - next is not defined {
  "url": "/api/users/me/settings",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:36:26 [error]: 500 - next is not defined {
  "url": "/api/users/me/settings",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:39:50 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:39:52 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:39:53 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:39:54 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:39:56 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:39:56 [error]: 500 - next is not defined {
  "url": "/api/calendrier",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:40:23 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:40:25 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:40:26 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:41:26 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:41:30 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:41:31 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:41:34 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:45:35 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:45:38 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:45:38 [error]: 500 - next is not defined {
  "url": "/api/calendrier",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:53:44 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 18:53:44 [error]: 500 - next is not defined {
  "url": "/api/calendrier",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 19:42:12 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 19:42:32 [error]: 500 - next is not defined {
  "url": "/api/conversations",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 19:42:37 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 19:43:08 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-05 19:43:09 [error]: 404 - Ressource non trouvée {
  "url": "/.well-known/appspecific/com.chrome.devtools.json",
  "method": "GET",
  "ip": "::1"
}
2025-09-08 11:32:28 [error]: Uncaught Exception: naissanceRoutes is not defined {
  "stack": "ReferenceError: naissanceRoutes is not defined\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:262:28)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "ReferenceError"
}
2025-09-08 11:33:22 [error]: Uncaught Exception: naissanceRoutes is not defined {
  "stack": "ReferenceError: naissanceRoutes is not defined\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:258:28)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "ReferenceError"
}
2025-09-08 11:37:19 [error]: Uncaught Exception: naissanceRoutes is not defined {
  "stack": "ReferenceError: naissanceRoutes is not defined\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:259:28)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "ReferenceError"
}
2025-09-08 11:46:44 [error]: Uncaught Exception: naissanceRoutes is not defined {
  "stack": "ReferenceError: naissanceRoutes is not defined\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:256:28)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "ReferenceError"
}
2025-09-08 12:18:19 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:475:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 16:54:24 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:428:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 16:54:45 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:428:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 18:10:13 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:428:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 18:11:01 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:428:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 20:42:30 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 21:08:30 [error]: Unhandled Rejection: {
  "reason": {},
  "stack": "TypeError: Cannot read properties of undefined (reading 'error')\n    at decesController.generateDecesPdf (/home/ben-chabaka/wenaklabs/controllers/decesController.js:125:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
2025-09-08 21:18:00 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 21:18:35 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 21:22:36 [error]: Unhandled Rejection: {
  "reason": {},
  "stack": "TypeError: Cannot read properties of undefined (reading 'error')\n    at decesController.generateDecesPdf (/home/ben-chabaka/wenaklabs/controllers/decesController.js:125:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
2025-09-08 21:26:13 [error]: Unhandled Rejection: {
  "reason": {},
  "stack": "TypeError: Cannot read properties of undefined (reading 'error')\n    at decesController.generateDecesPdf (/home/ben-chabaka/wenaklabs/controllers/decesController.js:125:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
2025-09-08 21:38:38 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 21:40:14 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 21:42:52 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:19:29 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:23:41 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:24:16 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:25:21 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:26:40 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:28:05 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:29:20 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:29:41 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:30:18 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:31:54 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:33:53 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:35:32 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-08 22:37:10 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:431:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
2025-09-10 09:58:40 [error]: Uncaught Exception: listen EADDRINUSE: address already in use :::3000 {
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1908:16)\n    at listenInCluster (node:net:1965:12)\n    at Server.listen (node:net:2067:7)\n    at Object.<anonymous> (/home/ben-chabaka/wenaklabs/app.js:444:8)\n    at Module._compile (node:internal/modules/cjs/loader:1529:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)\n    at Module.load (node:internal/modules/cjs/loader:1275:32)\n    at Module._load (node:internal/modules/cjs/loader:1096:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:164:12)\n    at node:internal/main/run_main_module:28:49",
  "name": "Error"
}
